app/views/templates/_related_news_articles.html.haml

```diff
@@ -0,0 +1,12 @@
+%aside.sidebar
+  %h2 Related news
+  %ul
+    %li
+      %a{:href => "#"}
+        A related article here
+    %li
+      %a{:href => "#"}
+        A related article here
+    %li
+      %a{:href => "#"}
+        A related article here
```

Comment:
We can probably make this real content, if we base this on the last 5 news articles in chronological order from the associated topics.

We have already implemented this kind of query for news filtering - it might be possible to abstract the code that was written there to take advantage of here.